Misc/NEWS.d/next/Core_and_Builtins/2024-12-17-18-20-37.gh-issue-128035.JwqHdB.rst

```diff
@@ -0,0 +1 @@
+TLSv1.3 post-handshake client authentication (PHA), often referred to as "mutual TLS" or "mTLS", allows TLS servers to authenticate client identities using digital certificates. This commit exposes a boolean property ``ssl.HAS_PHA`` to indicate whether the crypto library CPython is built against supports PHA, allowing python's test suite and consuming modules to branch accordingly.
```

Comment:
A shorter NEWS should be written. The NEWS is a message that users will see (it's in the changelog). Some suggestion:

```rst
Indicate through :data:`ssl.HAS_PHA` whether the :mod:`ssl` module supports TLSv1.3
post-handshake client authentication (PHA). Patch by YOURNAME.
```

In addition, you should add a What's New entry in `Doc/whatsnew/3.14.rst` indicating the additional constant. Usually, the same message as for the NEWS entry can be reused (check the other entries for the formatting).

Reply:
Thank you for the guidance. I've updated the news and whatsnew files accordingly.